_posts/2015-08-13-drupalcamp-asheville.md

```diff
@@ -15,7 +15,7 @@ drupal_planet_summary: |
 ---
 
 ## Drupalcamp Asheville, we're coming!
-Savas Labs will be represented at [Drupalcamp Asheville](http://www.drupalasheville.com) this year! [Anne](/company/anne-tomasevich/) and [Chris](/company/chris-russo/) are both presenting. Anne will present on Drupal 8 theming, roughly based off her [blog post about it](/2015/06/10/d8-theming-basics.html), a topic with very little reliable information out there in the wild, and Chris is presenting on his "personal git workflow, for everyone," which is also based off a [blog post he wrote recently](/2015/07/09/personal-git-workflow.html).
+Savas Labs will be represented at [Drupalcamp Asheville](http://www.drupalasheville.com) this year! [Anne](/company/) and [Chris](/company/chris-russo/) are both presenting. Anne will present on Drupal 8 theming, roughly based off her [blog post about it](/2015/06/10/d8-theming-basics.html), a topic with very little reliable information out there in the wild, and Chris is presenting on his "personal git workflow, for everyone," which is also based off a [blog post he wrote recently](/2015/07/09/personal-git-workflow.html).
```

Comment:
I think it's best to just remove the link to Anne rather than change to `/company/`, secondarily, you _could_ link to the partners section where Anne moved to, but again, would prefer the former.